src/common/map_coordinate.cc

```diff
@@ -0,0 +1,12 @@
+// BoB robotics includes
+#include "common/map_coordinate.h"
+
+namespace BoBRobotics {
+namespace MapCoordinate {
+const Ellipsoid WGS84::ellipsoid{units::length::meter_t(6378137), units::length::meter_t(6356752.314245)};
```

Comment:
Is this finally the end of ``NO_HEADER_DEFINITIONS``? :partying_face: I feel this instance was definitely my fault...

Reply:
Not quite:
```
alex @ ~/code/bob_robotics
 └─ gps_updates $ git grep -n NO_HEADER_DEFINITIONS -- :^docs
cmake/bob_robotics.cmake:268:    add_definitions(-DNO_HEADER_DEFINITIONS)
projects/snapshot_bot/image_input.cc:1:#define NO_HEADER_DEFINITIONS
projects/snapshot_bot/memory.cc:1:#define NO_HEADER_DEFINITIONS
```
:wink: